tests/stdlib/tdecodequery.nim

```diff
@@ -0,0 +1,10 @@
+import uri, sequtils
```

Comment:
move content to inside existing `tests/stdlib/turi.nim` since this tests std/uri

Reply:
-[ ] I could make a follow up PR since I also need to clean `cgi` module.

Reply:
=> https://github.com/nim-lang/Nim/pull/16500